### PR TITLE
[VEN-1024]Added comments for the setMarketSupplyCaps and setMarketBorrowCaps.

### PR DESCRIPTION
**Pull request by defcon022**
## Description

Resolves - VEN-1024
- Added comments for the setMarketSupplyCaps and setMarketBorrowCaps.

## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

File: contracts/Comptroller.sol [hunk 1]
```diff
@@ -842,6 +842,7 @@ contract Comptroller is
      * @notice Set the given borrow caps for the given vToken markets. Borrowing that brings total borrows to or above borrow cap will revert.
      * @dev This function is restricted by the AccessControlManager
      * @dev A borrow cap of -1 corresponds to unlimited borrowing.
+     * @dev We can not restrict the cap to the borrow limit as the borrow can be lower than the restricted cap. we want the system to operate on a lower borrow.
```
Comment (chechu) [inline contracts/Comptroller.sol:845]:
```suggestion
     * @dev Borrow caps smaller than the current total borrows are accepted. This way, new borrows will not be allowed until the total borrows amount goes below the new borrow cap
```

File: contracts/Comptroller.sol [hunk 2]
```diff
@@ -866,6 +867,7 @@ contract Comptroller is
      * @notice Set the given supply caps for the given vToken markets. Supply that brings total Supply to or above supply cap will revert.
      * @dev This function is restricted by the AccessControlManager
      * @dev A supply cap of -1 corresponds to unlimited supply.
+     * @dev We can not restrict the cap to the supply limit as the supply can be lower than the restricted cap. we want the system to operate on a lower supply.
```
Comment (chechu) [inline contracts/Comptroller.sol:870]:
```suggestion
     * @dev Supply caps smaller than the current total supplies are accepted. This way, new supplies will not be allowed until the total supplies amount goes below the new supply cap
```

Comment (defcon022) [reply contracts/Comptroller.sol]: Fixed.

File: contracts/Comptroller.sol (hunk 1 above)
Comment (defcon022) [reply contracts/Comptroller.sol]: Fixed.

**Review by chechu — APPROVED** [review]

**Pull request merged**